standalone/pherry/src/msg_sync.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
 use phaxt::subxt::tx::Signer as _;
 
 pub use tokio::sync::mpsc::{channel, Receiver, Sender};
+use crate::types::subxt::error::RpcError;
```

Comment:
For what?

Reply:
My bad, this not work
Error change may relates to https://github.com/paritytech/subxt/pull/694